### PR TITLE
Access csv files

### PR DESCRIPTION
**Pull request by jdcarey128**
This PR: 
- structures class to allow access to the csv files as ruby instance variables

### CODE REVIEW

**Review by dunlapww — APPROVED** [review]
Looks good Joshua!

**Comment by hopesgit** [comment]
Be sure not to keep those coverage files.

**Review by hopesgit — APPROVED** [review]

File: runner.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+team_path = './data/teams_test.csv'
+game_teams_path = './data/game_teams_test.csv'
+
+locations = {
+  games: game_path,
+  teams: team_path,
+  game_teams: game_teams_path
+}
+
+stattracker = StatTracker.from_csv(locations)
```
Comment (jakeheft) [inline runner.rb]: - change stattracker to stat_tracker (per Ruby convention)
- Do we want to keep stat_tracker with the regular data files, but include a stat_tracker_test as well so we can access both types of data?

**Review by jakeheft — APPROVED** [review]
All changes look good now!

**Pull request merged**